# single feature match.  Contains two points
uint64 queryId
uint64 trainId
float64 distance
uint64 imgId
